src/renderer/src/routes/(main)/app-store/[slug]/+page.svelte

```diff
@@ -55,13 +61,22 @@
 	const handleError = (error: unknown, versionEntity?: Entity<AppVersionEntry>) => {
 		console.error(error);
 		loadingString = '';
-		const errorMessage = getErrorMessage(error);
+		let errorMessage = getErrorMessage(error);
 		if (errorMessage === APP_NAME_EXISTS_ERROR && versionEntity) {
 			toastStore.trigger({
 				message: $i18n.t(errorMessage)
 			});
 			return installLogic(versionEntity);
 		}
+		if (errorMessage.includes('No available peer host found.')) {
+			errorMessage = NO_AVAILABLE_PEER_HOSTS_ERROR;
+		} else if (errorMessage.includes('Request timed out in 60000 ms: call_zome')) {
```

Comment:
Perhaps it's safer to just look for `Request timed out in`, in case the interval changes.

Reply:
Agreed and a good opportunity to re-introduce some of the vagueness that we had to let go of above ;)

Reply:
https://github.com/holochain/launcher/pull/248/commits/1f35a2dc8ff4909156f11488091c676283e08685